openmetadata-ui/src/main/resources/ui/src/components/TestSuitePipelineTab/TestSuitePipelineTab.component.tsx

```diff
@@ -69,7 +69,7 @@ const TestSuitePipelineTab = () => {
   const [isConfirmationModalOpen, setIsConfirmationModalOpen] = useState(false);
   const [currTriggerId, setCurrTriggerId] = useState({ id: '', state: '' });
   const [currDeployId, setCurrDeployId] = useState({ id: '', state: '' });
-  const [isAirflowRunning, setIsAirflowRunning] = useState(false);
+  const [isAirflowRunning, setIsAirflowRunning] = useState(true);
```

Comment:
Why we are making it default true?

Reply:
We are showing connect to airflow docs when this state is false, and we fetch the status airflow on the page load, so the issue we were facing was as flickering between the docs placeholder and actual data when the airflow is connected!
so I have added the default value as true so it won't show the docs placeholder and on page load, we are fetching airflow status it will update the appropriate airflow status and UI will act accordingly.
We also have one approach suggested by @Sachin-chaurasiya is to maintain different states for airflow API load
What's your opinion here?